### PR TITLE
Being humble, added apps, fixes #21

### LINKED ISSUE #21: Apps support.
Support querying and finding information on apps.

### PR DESCRIPTION
**Pull request by aandis**

### CODE REVIEW

**Comment by rohitpaulk** [comment]
Testsssssssssss. Should be very similar to the Movie ones, think you can get it done?

**Comment by aandis** [comment]
Yeah. I was gonna do it. Can you look at other pending prs and #10 and #14 ?

File: app/controllers/api/android_apps_controller.rb [hunk 1]
```diff
@@ -12,14 +12,14 @@ def index
...
-    render plain: "Send me an array, dumbass", status: 400 and return unless params[:apps].is_a?(Array)
+    render plain: "Sir, please give array", status: 400 and return unless params[:apps].is_a?(Array)
 
     updated_apps = @user.update_apps(params['apps'])
     render :json => updated_apps.to_json
   end
 
   def batch_delete
-    render plain: "Send me a uid param, dumbass!", status: 400 and return unless params[:uid]
+    render plain: "Sir, pleasee give uid param. Please sir.", status: 400 and return unless params[:uid]
```
Comment (aandis) [inline app/controllers/api/android_apps_controller.rb:23]: @rohitpaulk us back then writing apis.

Comment (rohitpaulk) [reply app/controllers/api/android_apps_controller.rb:23]: haha

Comment (rohitpaulk) [reply app/controllers/api/android_apps_controller.rb:23]: I'd make this `Saar` to adhere to our coding guidelines

**Pull request merged**